apps/cowswap-frontend/src/modules/twap/state/twapOrdersListAtom.ts

```diff
@@ -14,11 +16,7 @@ import { updateTwapOrdersList } from '../utils/updateTwapOrdersList'
 
 export type TwapOrdersList = { [key: string]: TwapOrderItem }
 
-export const twapOrdersAtom = atomWithStorage<TwapOrdersList>(
-  'twap-orders-list:v1',
-  {},
-  createJSONStorage(() => localStorage)
-)
+export const twapOrdersAtom = atomWithStorage<TwapOrdersList>('twap-orders-list:v1', {}, getJotaiIsolatedStorage())
```

Comment:
I think the orders' list should be synced, no?

Reply:
No, https://github.com/cowprotocol/cowswap/pull/2903